### PR TITLE
must-gather: call gather_common_ceph_resources with proper args

### PR DESCRIPTION
**Pull request by agarwal-mudit**
Signed-off-by: Mudit Agarwal <muagarwa@redhat.com>

### CODE REVIEW

**Comment by agarwal-mudit** [comment]
For https://bugzilla.redhat.com/show_bug.cgi?id=1982960

**Review by parthdhanjal — COMMENTED** [review]
LGTM!

File: must-gather/collection-scripts/gather_ceph_resources [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-gather_common_ceph_resources "${BASE_COLLECTION_PATH}" "${SINCE_TIME}"
+gather_common_ceph_resources "$1" "$2"
 
 # Expect base collection path as an argument
 BASE_COLLECTION_PATH=$1
```
Comment (gobindadas) [inline must-gather/collection-scripts/gather_ceph_resources:6]: It woud be better to first collect Path and time like:
BASE_COLLECTION_PATH=$1
SINCE_TIME=$2
Then declare common resources:  gather_common_ceph_resources "${BASE_COLLECTION_PATH}" "${SINCE_TIME}"

**Review by gobindadas — APPROVED** [review]

**Review by obnoxxx — APPROVED** [review]
/lgtm

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**